Small improvements in documentation. (#223)
Microsoft-community/Community-website: src/wiki/ventoy.md

## Downloading ISO files
### Windows

> To download a Windows ISO file, refer to the [Downloading Windows](./downloading-windows.md) page for only wanting to download the image of Windows via the Media Creation Tool.

Reviewer: ```suggestion
To download a Windows ISO file, refer to the [Downloading Windows](./downloading-windows.md) page to learn how to download the Windows image via the Media Creation Tool.
```

- **Clonezilla** (Disk imaging and cloning): https://clonezilla.org/downloads/download.php?branch=stable

::: tip Note
You can read more about reinstalling Windows on the [Back Up Data](/wiki/backup#alternative-imaging-method-ventoy) wiki page.
:::

> - **Gparted** (Disk partitioning): https://gparted.org/download.php

Reviewer: You only need one empty line
